docs/source/examples/checks/vision/distribution/train_test_label_drift.ipynb

```diff
@@ -146,7 +146,7 @@
     {
```

Comment:
<ol><li>What is the units for the bbox area?</li><li>Set lower ylim in discrete probability plots. </li><li>put number of samples per class first</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>